docker-compose.yml

```diff
@@ -55,6 +57,8 @@ services:
 
   memcached:
     image: memcached:1.4
+    # Remove this once we upgrade to a version that provides multi-platform images
```

Comment:
We're _only_ removing this `platform` once there's an upgrade to a new version?  Or does this apply to each service defined in this file?

Reply:
Technically it would be true for each, but `memcached` is the one where it's most likely to happen soon because newer versions that are multi-platform already exist, where as, as we proved in the issue, it's not trivial to make that change for our images.

Reply:
Yes, that's why I decided not to add this comment to the other sections. I can do that though if you prefer.